client/src/components/common/Card.svelte

```diff
@@ -0,0 +1,108 @@
+<script>
+  import { slide, blur } from 'svelte/transition';
+  export let recipe;
+  export let rating;
+
+  function extendOptions(e) {
+    let circles = Array.prototype.slice.call(e.target.parentNode.parentNode.querySelectorAll('.circle'));
+    circles.forEach(child => {
```

Comment:
same as with event listeners we don't need or want to do this. This can easily be done like so
```js
// <script>
  let extended = false;
  function toggleExtended() {
    extended = !extended;
  }
// </script>

// <main>
    <div class="circle info {extended && 'extend'}">...</div>
    <div class="circle rating {extended && 'extend'}">...</div>
    <div class="circle time {extended && 'extend'}">...</div>
// </main>
```